src/app/features/task-repeat-cfg/store/task-repeat-cfg.reducer.spec.ts

```diff
@@ -195,71 +224,275 @@ describe('selectTaskRepeatCfgsDueOnDay', () => {
       expect(resultIds).toEqual(['R1']);
     });
 
-    it('should work correctly for a week', () => {
```

Comment:
Did you remove this, because you think it's an invalid test?

Reply:
Yes and no. I originally expanded on this test and then added my own. But… it's now obsolete.
As the commit message of 3f65adeead3f678723eec602c785b3d6c59913ba states:
> This test lacks correct lastTaskCreation attributes and is supervised/covered by another test.
It may be re-added later.